### PR TITLE
setup to do list with webpack

### PR DESCRIPTION
**Pull request by TayyabKM**
### The following actions were taken:

- Set up the project with webpack.
- Pushed array objects dynamically through javascript.
- Set up files and passed Linter errors.
- Set up to-do list HTML structure and basic CSS

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+* {
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+  margin: 0;
+  padding: 0;
+}
```
Comment (aimalamiri) [inline src/style.css:8]: Good job 👌but as it's mentioned in the project requirements you're supposed to design your to-do list similar to the one in this [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg&ab_channel=EmiliaAndrzejewska). So please make sure to design your application using CSS.

image

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+  ],
+  // output: {
+  //   filename: '[main].bundle.js',
+  //   path: path.resolve(__dirname, 'dist'),
+  //   clean: true,
+  // },
```
Comment (aimalamiri) [inline webpack.config.js:16]: It's part of the best practices to not keep commented code blocks in your codebase. Kindly make sure to remove it. It'll help you to have a cleaner repository.

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @TayyabKM, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlmostThereReachGIF

## Highlights

- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [x] It's highly recommended to name your branch which describes the task like `configure-webpack`, or `add-feature-X` instead of `feature-branch`. 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 1 above)
Comment (TayyabKM) [reply src/style.css:8]: Hey @aimalamiri , If i rename the branch it will close the pull request, is that okay?

Comment (aimalamiri) [reply src/style.css:8]: No, you don't need to rename the branch for this pull request, that's why I put that request under the optional section. But I forget to add the `[Optional]` Prefix. Sorry about that.

Comment (TayyabKM) [reply src/style.css:8]: It's all right, is the styling css also an optional change?

Comment (aimalamiri) [reply src/style.css:8]: Good question! It's a required task, as you can see I pasted a screenshot from the requirements.

Comment (mckent05) [reply src/style.css:8]: - Please style your project to look like the wireframe, just as the previous reviewer suggested. Only comments with the `OPTIONAL` tag prefix can be considered optional changes. Please fix this issue 😄 

Your project:
wwdwd

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @TayyabKM, 

#STATUS: CHANGES REQUIRED ♻️ ♻️ 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
almost there



 ## Highlights
- All linter checks are passing ✔️ ✔️ 
- PR summary and title are descriptive 👏 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your code.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**